extensions/amp-iframe/0.1/amp-iframe.js

```diff
@@ -127,10 +143,10 @@ class AmpIframe extends AMP.BaseElement {
     iframe.width = getLengthNumeral(width);
     iframe.height = getLengthNumeral(height);
     iframe.name = 'amp_iframe' + count++;
-    iframe.onload = function() {
-      // Chrome does not reflect the iframe readystate.
-      this.readyState = 'complete';
-    };
+
+    if (this.isClickToPlay_) {
+      iframe.style.zIndex = -1;
```

Comment:
the placeholder's zindex is 1 !important - not sure if we should with override this and change the placeholder's behavior. I dont like the idea if overriding an !important with another